The upstream patch's hash changed. Everything looks kosher so just update the pinned hash.

diff --git a/pkgs/development/compilers/gcc/ng/common/libgcc/default.nix b/pkgs/development/compilers/gcc/ng/common/libgcc/default.nix
index 1f130f95dae2..7799119f688b 100644
--- a/pkgs/development/compilers/gcc/ng/common/libgcc/default.nix
+++ b/pkgs/development/compilers/gcc/ng/common/libgcc/default.nix
@@ -201,7 +201,7 @@ stdenv.mkDerivation (finalAttrs: {
       (fetchpatch {
         name = "regular-libdir-includedir.patch";
         url = "https://inbox.sourceware.org/gcc-patches/20250717174911.1536129-1-git@JohnEricson.me/raw";
-        hash = "sha256-Cn7rvg1FI7H/26GzSe4pv5VW/gvwbwGqivAqEeawkwk=";
+        hash = "sha256-unPDAwW0F6ZSjSNLRiO3zcTHPKZKflK+2Yu8SYX1e8Q=";
       })
       (getVersionFile "libgcc/force-regular-dirs.patch")
     ];
diff --git a/pkgs/os-specific/linux/minimal-bootstrap/gcc/common.nix b/pkgs/os-specific/linux/minimal-bootstrap/gcc/common.nix
index 6eaf611e1b40..d4ac9e2d1310 100644
--- a/pkgs/os-specific/linux/minimal-bootstrap/gcc/common.nix
+++ b/pkgs/os-specific/linux/minimal-bootstrap/gcc/common.nix
@@ -16,7 +16,7 @@ let
     (fetchurl {
       name = "regular-libdir-includedir.patch";
       url = "https://inbox.sourceware.org/gcc-patches/20250717174911.1536129-1-git@JohnEricson.me/raw";
-      hash = "sha256-Cn7rvg1FI7H/26GzSe4pv5VW/gvwbwGqivAqEeawkwk=";
+      hash = "sha256-unPDAwW0F6ZSjSNLRiO3zcTHPKZKflK+2Yu8SYX1e8Q=";
     })
     ./configure-skip-target-libs-hooks.patch
   ];
